docs/docs/08.1-more-about-refs.md

```diff
@@ -88,28 +88,28 @@ React also supports using a string (instead of a callback) as a ref prop on any
 In order to get a reference to a React component, you can either use `this` to get the current React component, or you can use a ref to get a reference to a component you own. They work like this:
 
 ```javascript
-var MyComponent = React.createClass({
-  handleClick: function() {
+const MyComponent = (textInput) => {
```

Comment:
This will need to be an ES6 class, because the ref needs to be attached to the this object.